instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/JarServiceNameDetector.java

```diff
@@ -9,6 +9,7 @@
 
 import com.google.auto.service.AutoService;
 import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.incubator.config.DeclarativeConfigProperties;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
 import io.opentelemetry.sdk.autoconfigure.spi.ResourceProvider;
 import io.opentelemetry.sdk.autoconfigure.spi.internal.ConditionalResourceProvider;
```

Comment:
can you open an SDK issue about whether this can/should be made public also? thanks

Reply:
it is public - not in an internal package. `incubator` is used because it's not stable yet